prover/Cargo.toml

```diff
@@ -15,7 +15,7 @@ rust-version = "1.75"
 [features]
 concurrent = ["processor/concurrent", "std", "winter-prover/concurrent"]
 default = ["std"]
-metal = ["dep:ministark-gpu", "dep:elsa", "dep:pollster", "concurrent", "std"]
+metal = ["dep:miden-gpu", "dep:elsa", "dep:pollster"]
```

Comment:
Just a mistake. I confused my self because i remembered removing std and concurrent features from miden-gpu, and just removed them here without thinking.. I reverted the change